lib/voxpupuli/release/rake_tasks.rb

```diff
@@ -52,3 +52,116 @@
     fail "Unable to find a CHANGELOG.md entry for the #{v} release."
   end
 end
+
+desc "Prepare a release"
+task "release:prepare" do
+  v = Blacksmith::Modulefile.new.version
+  Rake::Task["release:porcelain:changelog"].invoke
+  Rake::Task["strings:generate:reference"].invoke if File.exist?('REFERENCE.md')
+  puts <<~MESSAGE
+
+    Please review these changes and commit them to a new branch:
+
+        git checkout -b release-#{v}
+        git commit -m "Release #{v}"
+
+    Then open a Pull-Request and wait for it to be reviewed and merged).
+  MESSAGE
+end
+
+begin
+  require 'github_changelog_generator/task'
+  require 'puppet_blacksmith'
+rescue LoadError
+  desc "Dummy"
+  task "release:porcelain:changelog" do
+    puts "Skipping CHANGELOG.md generation.  Ensure github_changelog_generator is present if you expected it to be generated."
```

Comment:
Shouldn't this hard fail instead?

Reply:
I think we wanted to avoid a hard-failure for projects using our tooling but not hosted on GitHub and where GCG is not usable.  I have never tested this however and maybe it is not functional at all…  For now I would prefer to keep that this way if you don't mind.